packages/ERTP/test/unitTests/test-inputValidation.js

```diff
@@ -114,8 +113,7 @@ test('makeIssuerKit malicious displayInfo', async t => {
         'bad displayInfo',
       ),
     {
-      message:
-        '"displayInfo" "bad displayInfo" must be a pass-by-copy record, not "string"',
+      message: /"bad displayInfo" - Must have shape of base: "copyRecord"/,
```

Comment:
This only mentions 'displayInfo' because it was in the payload. Without it, this message would be hard to track down.

Reply:
This one does suggest an easy enhancement I'd be happy to make in this PR, even though it would require a coordinated change to the pattern matcher: extend `fit` to take a third argument, which is a label string, much like the first argument to `Far` or `makeScalarMapStore`, that can be included in the thrown error as an additional clue about what is being tested or what the point is of testing it. In this case, `fit(..., ..., 'displayInfo')` would then, perhaps, result in prepending `displayInfo:` to the diagnostic string.

But, since it does involve a coordinated change to the pattern matchers, I'd still rather file a ticket and then do these improvements incrementally over time.

Reply:
Nah. I just gave it a shot and the easy way I had in mind wasn't so easy.